go/apps/api/routes/v2_identities_get_identity/400_test.go

```diff
@@ -37,8 +37,8 @@ func TestBadRequests(t *testing.T) {
 		require.Equal(t, "https://unkey.com/docs/api-reference/errors-v2/unkey/application/invalid_input", res.Body.Error.Type)
 		require.Equal(t, "POST request body for '/v2/identities.getIdentity' failed to validate schema", res.Body.Error.Detail)
 		require.GreaterOrEqual(t, len(res.Body.Error.Errors), 1)
-		require.Equal(t, "/oneOf/0/required", res.Body.Error.Errors[0].Location)
-		require.Equal(t, "missing property 'identityId'", res.Body.Error.Errors[0].Message)
+		require.Equal(t, "/oneOf", res.Body.Error.Errors[0].Location)
+		require.Equal(t, "'oneOf' failed, none matched", res.Body.Error.Errors[0].Message)
```

Comment:
for later, we should figure out how to provide better and more helpful error messages here, if I'd get this response from an API, I'd be mad

@coderabbitai please create an issue for this